rclcpp_action/include/rclcpp_action/client_goal_handle_impl.hpp

```diff
@@ -71,6 +73,9 @@ ClientGoalHandle<ActionT>::set_result(const WrappedResult & wrapped_result)
   std::lock_guard<std::mutex> guard(handle_mutex_);
   status_ = static_cast<int8_t>(wrapped_result.code);
   result_promise_.set_value(wrapped_result);
+  if (nullptr != result_callback_) {
```

Comment:
@jacobperron nit: why not
```suggestion
  if (result_callback_) {
```
like it's done for the `goal_response_callback` above?

Reply:
eeed6f5